packages/core/src/lib/keep_alive_manager.ts

```diff
@@ -105,4 +100,39 @@ export class KeepAliveManager {
     this.pingKeepAliveTimers.clear();
     this.relayKeepAliveTimers.clear();
   }
+
+  private sendRelayPings(
```

Comment:
Let's rename to `scheduleRelayPings` or `setRelayPingsOninterval` or something along those lines.